docs/LogGuide.md

```diff
@@ -2068,6 +2082,48 @@ ACT log lines are blank for this type.
 
 These are lines emitted directly by the ffxiv plugin when something goes wrong.
 
+## OverlayPlugin Log Lines
+
+If you are using OverlayPlugin,
+it will emit extra log lines that are not part of the ffxiv plugin.
+The ids of these lines start at 256 and go up.
+Any id between 0-255 is reserved for the ffxiv plugin.
+
+<a name="line256"></a>
+
+### Line 256 (0x100): LineRegistration
+
+This line is emitted into logs when any custom logs are registered with OverlayPlugin.
+This is so that it is obvious which log lines and versions to expect for a given log file.
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=LineRegistration&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+256|[timestamp]|[id]|[source]|[version]
+
+ACT Log Line Structure:
+[timestamp] 256 100:[id]:[source]:[version]
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+256|2022-10-02T10:15:31.5635165-07:00|257|OverlayPlugin|MapEffect|1|594b867ee2199369
+256|2022-10-02T10:15:31.5645159-07:00|258|OverlayPlugin|FateDirector|1|102a238b2495bfd0
+256|2022-10-02T10:15:31.5655143-07:00|259|OverlayPlugin|CEDirector|1|35546b48906c41b2
+
+ACT Log Line Examples:
+[10:15:31.563] 256 100:257:OverlayPlugin:MapEffect:1
+[10:15:31.564] 256 100:258:OverlayPlugin:FateDirector:1
+[10:15:31.565] 256 100:259:OverlayPlugin:CEDirector:1
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=LineRegistration&lang=en-US) -->
+
 <a name="line257"></a>
 
 ### Line 257 (0x101): MapEffect
```

Comment:
I know that this was added in a previous commit, but I believe it has been determined that `id` is more of a flags/bitmask field, rather than a raw uint.